src/Hello/Hello.cs

```diff
@@ -34,7 +34,10 @@
 // Create a simple console logger factory and configure the log level for category IceRpc.
 using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
     builder
-    .AddSimpleConsole()
+    .AddSimpleConsole(options =>
+        {
+            options.TimestampFormat = "[yyyy/MM/dd HH:mm:ss] ";
```

Comment:
The extra space on the end is required and not a typo.

Reply:
What about including milliseconds in this timestamp?

https://learn.microsoft.com/en-us/dotnet/standard/base-types/how-to-display-milliseconds-in-date-and-time-values